### ISSUE TITLE
Train Caffe Model task failed with error code -11

### ISSUE THREAD

**Issue by CrowMagnumMan** [NVIDIA/DIGITS#2010]
I'm trying to train the DetectNet-COCO-Bottle example.  My training server is 2 Titan XPs in SLI that has successfully trained the other examples (really fast).

When I try to train the coco-bottle model, I immediately get "[ERROR] Train Caffe Model task failed with error code -11" on the console and "Layer's types are Ftype:FLOAT Btype:FLOAT Fmath:FLOAT Bmath:FLOAT
Importing Python module 'caffe.layers.detectnet.clustering'" on the Digits Train Caffe Model window.

Any clues?

**Comment by mantianwuming** [comment]
I met the same problem, too. Did you solve it?

**Comment by CrowMagnumMan** [comment]
Nope, not yet.  I haven't spent much time on it.  Was hoping for an answer....  I'll mess with it again this weekend if I don't get any solutions here.

**Comment by mantianwuming** [comment]
I found many methods but no one can truely solve it. I'll try again and if I'm lucky enough and get the answer, I'll write it here. Thanks for your reply and, as you can see, I am not good at English.

**Comment by CrowMagnumMan** [comment]
Your English is fine.  The saga continues, and it just keeps getting weirder.  The system that it fails on has two Titan XPs and is running Ubuntu 16.04 LTS.  This system rocks on everything else DNN.
However, I just ran it on a system that has only one Titan XP and is running Ubuntu 14.04 LTS.  It worked fine, took around 4.5 hours.
So the DetectNet-COCO-Bottle example trains fine on one GPU with older Ubuntu, but fails on two GPUs with newer Ubuntu...

**Comment by mantianwuming** [comment]
Thanks and I have just one GPU, it's 1080 Ti. I will try more to solve it.

**Comment by galdalali** [comment]
@CrowMagnumMan, did you found a solution? I have same issue.

**Comment by CrowMagnumMan** [comment]
Nope, never could get it to work with the two GPU/Ubuntu 16.04 system.  It either doesn't like two GPUs or doesn't like Ubuntu 16.04.  My guess is the latter.

**Comment by felixmejia** [comment]
Hi, I get this error, somebody can solve this error. Please

**Comment by CrowMagnumMan** [comment]
What is your operating system and version?  Are you running one or multiple GPUs and what 'is it'  || 'are they'?

**Comment by xiaomaozhou26** [comment]
pip install protocbuf==3.1.0.post1 solved my problem

**Comment by xiaomaozhou26** [comment]
pip install protobuf==3.1.0.post1 solved my problem
